packages/p2p/src/stores/my-ads-store.js

```diff
@@ -391,7 +389,10 @@ export default class MyAdsStore extends BaseStore {
                     setSubmitting(false);
                     this.setApiErrorCode(response.error.code);
                     this.setEditAdFormError(response.error.message);
-                    this.setIsEditAdErrorModalVisible(true);
+                    general_store.showModal({
+                        key: 'AdEditErrorModal',
+                        props: {},
```

Comment:
`question`: if we don't pass props and just pass the key, would it still be fine?

Reply:
yes. updated.